_posts/2020-09-14-Introduction-to-API-Management.md

```diff
@@ -0,0 +1,292 @@
+
+In our current digital world where everyone is creating APIs, we should look at how we can manage and control them.
+
+
+Most of us know that we not only have to build the API but foresee additional responsibilities. 
+Some are just a necessary evil, like adding cross-cutting concerns.  
+Keeping an overview of all available API's can become a struggle these days.
+
+
+        
+Well, instead of trying to reinvent the wheel, I searched for the best solution and found that an API Management tool can solve a lot of those problems and add extra value within your landscape.
+
+An API management tool helps you in managing your APIs, setting the responsibilities on the right level. 
+
+
+You can add plenty additional functionalities, but most of all this happens in one tool!
+That’s one of the reasons it is important for your business to implement such a tool in your organization.
+        
+                
+## Investigation
+
+### API Management or API Gateway?
+The terms API management and API gateway are often used interchangeably, which is not correct.
+An API management tool refers to the application or overall solution of managing APIs.
+This mostly indicates that they are talking about the application and its features.
+A management tool can include one or multiple API Gateways, depending on the solution needed in the organization.
+An API gateway is actually a middleware that is placed in front of your services an act as a central point to administrate, route and secure your services.
+
+
+### API landscape without an API management tool
+Before going into the details, let’s take a look on how most of the current API landscapes are build and how we can improve those with an API management tool.  
+Without an API manager, your APIs will be the direct integration point to the business domain data.
+Any integration of for example security, needs to be implemented in your API. This sometimes leads to duplicate code and the fact that your API is not only responsible for exposing the Data, but also for implementing other cross cutting functionalities.
+	     
+<img src="/img/2020-09-14-Introduction-to-API-Management/WithoutApiManagementTool2.jpg" alt="API landscape without an API management tool" width="350" height="200" class="image fit">
+
+### API landscape with an API management tool
+This all changes when we implement an API Management tool.
+
+<img src="/img/2020-09-14-Introduction-to-API-Management/WithApiManagementTool2.jpg" alt="API landscape with an API management tool" width="500" height="350" class="image fit" style="vertical-align:middle;margin-left:2%"/>       
+
+In this situation an API management tool (API Gateway) is deployed between the client and the APIs.
+All request first have to pass the API management tool, before they are forwarded to the right service.
+If the Gateway is installed in the same organizational structure, it’s not required to have your API’s implementing additional cross cutting functionalities.

```

Comment:
Debatable, some services might need stricter constraints, the gateway can set a baseline here. 
Cross-cutting concerns are very broad so it's maybe better to rephrase this

Reply:
It was a proposal from Ken to use Cross cutting func. 
Initially I used a small summary of the most important parts.
You will see that it's coming back multiple times in the blog post.
I didn't find a better way to rephrase it. do you maybe have an suggestion?

Reply:
Maybe in the form of: it will be possible to move most of the cross-cutting concerns to the API Gateway

Reply:
Let me know what you think about the rephrased section (after I pushed it :) )